disbatch/README.md

```diff
@@ -1,9 +1,22 @@
-This directory include the tools that make use of disBatch.
+This directory includes that enable users to use [disbatch](https://github.com/flatironinstitute/disBatch), which is designed to submit a large number of tasks to slurm.
 
-"python createDisBTasks.py" will generate a disBatch taskfile "disB_tasks_all" that includes all the input files under /mnt/ceph/users/humanbase/data/meta/datasets/.
+First, we generate a disBatch taskfile by executing
 
-Inside disBatch taskfile, "disB_run.sh" is used to run with an input file. In turn, "disB_run.sh" sets up environment and calls "python run_pipeline.py" with parameters.
+`python createDisBTasks.py`
 
-". ./disB_submit.sh" will finally submit the disBatch job to Slurm.
+The taskfile `disB_tasks_all` is generated under the current directory. 
+Each line in the file is corresponding to a slurm task. 
+We have a task for each non-empty data file under directory `INPUT_DIR`, which is defined and can be modified inside createDisBTasks.py. 
+Each line in `disB_tasks_all` is a command executing `disB_run.sh` such as,
 
-The results are saved under /mnt/ceph/users/yliu/projects/MetaSRA-pipeline/data/
+`./disB_run.sh SRP058038 -o ~/ceph/projects/MetaSRA-pipeline/data/ /mnt/ceph/users/humanbase/data/meta/datasets/SRP058038.json`,
```

Comment:
Doesn't this assume `ceph` is symlinked in the person's home dir (e.g. mine is not)? Should clarify in the README if so.

Reply:
Changed README.md